libtiledbvcf/src/write/writer.cc

```diff
@@ -429,6 +431,12 @@ std::pair<uint64_t, uint64_t> Writer::ingest_samples(
             throw std::runtime_error(
                 "Error submitting TileDB write query; unexpected query "
                 "status.");
+
+          if (ingestion_params_.verbose)
+            std::cout << "Writing " << worker->records_buffered()
+                      << " for contig " << worker->region().seq_name
+                      << " (task " << i << " / " << tasks.size() << ")"
+                      << std::endl;
```

Comment:
How about something like "Recorded `x` cells for contig `y`" to be more inline with our export verbose messages (e.g., Processed 79133 cells in 0.153361 sec.). 

Also `i +1` for tasks?